src/__test__/ext.spec.js

```diff
@@ -25,21 +24,21 @@ describe('ext', () => {
 
         it('should return true for valid expression', () => {
           const isExpression = dimensionItems.visibilityCondition.isExpression(val);
-          expect(isExpression).to.be.true;
+          expect(isExpression).toBeTruthy();
```

Comment:
This is less strict than before. IMO, change to `toBe(true)`. Same for the other ones